README.md

```diff
@@ -1 +1,147 @@
+
+Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
+
+`git clone <your project link>`
+
+Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
+
+Step 2: Locate the folder where you cloned your project and open the `index.html` file in your browser to view your project.
+
+### Install
+
+Install this project with:
+
+1. install WebHint: npm install --save-dev hint@7.x
+2. install Stylelint: npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+3. install live Server from the extensions section on Vs Code Studio if using this code editor
+
+### Usage
+
+To run the project, execute the following command:
+open the cloned repository in your code editor
+
+### Run tests
+
+To run tests, run the following command:
+
+To run tests and check for errors:
+-After installing the required linter, npx stylelint "**/*.{css,scss}"
+-To fix CSS or SCSS linters error: npx stylelint "**/*.{css,scss}" --fix
+-Run your code on web browser
+
+### Deployment
+
+You can deploy this project using: github pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Author1**
+  
+- GitHub: [@Kidd254](https://github.com/Kidd254)
+- Twitter: [@lawrenc98789206](https://twitter.com/lawrenc98789206)
+- LinkedIn: [lawrence-kioko-972035240/](https://www.linkedin.com/in/lawrence-kioko-972035240/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly offer your support in terms of contributions. If you notice any issues you can raise them in the issues section, kindly. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Vote of thanks to Microverse for giving me an opportunity and ispiring me to complete this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Where is the functionality in this project?]**
+
+  - [this project is work in progress, more changes are likely to be added in the due course]
+
+- **[What is the aim of this project?]**
+
+  - [This project is meant to equip a learner with knowledge on correct github flow, and correct es linter configuration set up]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Currently your MIT license link is broken and is throwing a 404 error, Please add a License file to the root of your project with the appropriate content for MIT license and make sure the link isn't broken :+1: